### DISCUSSION TITLE
California: How to migrate to primefaces 14.0.0

### DISCUSSION THREAD

**Discussion by hichamglia** [primefaces/community#2025] [PrimeFaces Templates] [answered]
Hi,

I just migrated to primefaces 14.0.0 by changing the version in the dependency section of my project POM file. but I get the following : 
```
StandardWrapperValve[Faces Servlet]: Servlet.service() for servlet Faces Servlet threw exception
java.lang.ClassNotFoundException: org.primefaces.expression.SearchExpressionFacade
```

This project uses California-5.1.0 template.
Any help please!

**Comment by Rapster** [comment, marked as answer]
You might want to read this https://primefaces.github.io/primefaces/14_0_0/#/../migrationguide/14_0_0

**Reply by Rapster** [reply]
Pinging @melloware at it's premium template related

**Reply by hichamglia** [reply]
Thank you @Rapster, it is in fact explicitly mentioned in the migration guide
`org.primefaces.expression.SearchExpressionFacade removed in favor of org.primefaces.expression.SearchExpressionUtils`

Just found that SearchExpressionFacade is used in CaliforniaMenuRenderer
Do I have to request an update for California template or is there a procedure to make it my self?

**Reply by Rapster** [reply]
See https://github.com/orgs/primefaces/discussions/1806

**Comment by melloware** [comment]
@hichamglia if you send me a DM on the PrimeFaces Discord channel i can definitely help you upgrade it if you get stuck.

**Reply by hichamglia** [reply]
Thank you so much

**Reply by melloware** [reply]
we got the issue resolved and California upgraded to PF 14.0.0 successfully.
